sdk/go/channels.go

```diff
@@ -24,7 +24,7 @@ func (sdk mfSDK) CreateChannel(channel Channel, token string) (string, error) {
 		return "", ErrInvalidArgs
 	}
 
-	url := createURL(sdk.url, sdk.thingsPrefix, channelsEndpoint)
+	url := createURL(sdk.baseURL, sdk.thingsPrefix, channelsEndpoint)
```

Comment:
Have this been tested on both docker and local installation?

Reply:
This have been tested from CLI locally.